projects/components/src/table/table.component.ts

```diff
@@ -426,13 +425,6 @@ export class TableComponent
     }
   }
 
-  public ngAfterViewInit(): void {
```

Comment:
@anandtiwary did you test this for all forms of table - normal, expandable, and tree?